plugins/woocommerce/templates/cart/cart-totals.php

```diff
@@ -24,6 +24,8 @@
 
 	<h2><?php esc_html_e( 'Cart totals', 'woocommerce' ); ?></h2>
 
+	<?php do_action( 'woocommerce_before_cart_totals_table' ); ?>
```

Comment:
Hi! Could you add docblocks to both hooks? They should also include a `@version` tag, which can be set to `8.5.0`.